apps/ensadmin/src/app/@actions/gql/ponder/page.tsx

```diff
@@ -1,5 +1,5 @@
 import { CopyButton } from "@/components/ui/copy-button";
-import { preferredEnsNodeUrl } from "@/lib/env";
+import { defaultEnsNodeUrl } from "@/lib/env";
```

Comment:
Addressing feedback request:
> I would also suggest to rename `NEXT_PUBLIC_PREFERRED_ENSNODE_URL`
>
> The word "PREFERRED" seems like it would be better changed to "DEFAULT"